### ISSUE TITLE
[Accessibility] AutomationID property should not be empty for dropdown items under ImageAlign/TextAlign/CheckAlign/Dock in PropertyGrid

### ISSUE THREAD

**Issue by Olina-Zhang** [dotnet/winforms#7830]
### .NET version

.Net 8.0 SDK: 8.0.100-alpha.1.22469.15

### Did it work in .NET Framework?

Yes

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

In .Net 3.1, this issue cannot repro, and .Net 6.0 and 7.0 can repro

### Issue description

AutomationID property should not be empty for dropdown items under ImageAlign/TextAlign/CheckAlign/Anchor/Dock property in PropertyGrid

Inspect:
![image](https://user-images.githubusercontent.com/26474449/191923568-dd6a96bd-2c97-4007-8bf2-b9070d9687ee.png)

### Steps to reproduce

**Minimal repro:**
1. Create a winforms core app with PropertyGrid and RadioButton control.
2. Associate the Button with PropertyGrid as a selected object.
3. Build and run the core project.
4. Start Inspect tool.
5. Open the drop down list next to ImageAlign.
6. Focus on "Top Center" and observe AutomationID property in Inspect.

Tested app: 
[WinFormsApp13.zip](https://github.com/dotnet/winforms/files/9632177/WinFormsApp13.zip)

**Comment by MandiMan** [comment]
Verified with .NET SDK .NET 8.0.100-alpha.1.22552.10 build from main branch, this issue is fixed that AutomationID property is not empty for dropdown items under ImageAlign/TextAlign/CheckAlign/Anchor/Dock property in PropertyGrid.
![issue 7830](https://user-images.githubusercontent.com/108860782/199683444-50c13676-011b-4ce4-91c7-0a726c156b15.png)
